src/prototype_room_my.js

```diff
@@ -326,6 +326,57 @@ Room.prototype.executeRoom = function() {
     }
   }
 
+  let carryHelpInterval = config.carryHelpers.ticksUntilHelpCheck;
```

Comment:
I think we should extract it to a method.

And I would initialize the use variables first, like;

```
Memory.needEnergyRooms = Memory.needEnergyRooms || [];
this.memory.energyAvailableSum = this.memory.energyAvailableSum || 0;
...
```